src/Components/PrivateRoute.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Route, Navigate } from "react-router-dom";
+
+export const PrivateRoute = ({ ...props }) => {
+  const storage = localStorage?.getItem("QuizLoginUser");
+
+  if (storage !== null) {
+    const loginStatus = JSON.parse(storage);
+    if (loginStatus.isUserLogin === true) {
```

Comment:
I think `if (loginStatus.isUserLogin){...}` will work the same.

Reply:
Done